docs/get_started/tour.mdx

```diff
@@ -511,6 +596,21 @@ The `AppMain.java` file contains the definition of the endpoint that hosts the s
     The `app.py` file contains the definition of the endpoint that hosts the services.
 
 </TabItem>
+    <TabItem value="rust" label="Rust">
+        In `src/app` you will find the skeletons of the various services to help you start implementing the app.
+        For example:
+        ```rust checkout_service.rs
+        CODE_LOAD::https://raw.githubusercontent.com/restatedev/examples/refs/heads/rust_tour_split/tutorials/tour-of-restate-rust/src/part1/checkout_service.rs#checkout
+        ```
+
+        You implement your Rust Restate service by implementing a trait which contains the handlers.
+
+        Restate handlers have the Restate Context supplied as the first argument.
```

Comment:
Every time you refer to a type in the SDK, I think you should link the docs.rs here